reactioncommerce/reaction
# Use new CLI tool "propel" to deploy services to ECS

Impact: **minor**  
Type: **feature**

## Issue
We created a new separate AWS account to be used for staging environments. We also created a new CLI tool called propel to be used for deploying services to ECS clusters in a given environment. This PR pushes changes to CircleCI so that the new propel tool is used instead of ad-hoc shell scripts.

## Solution
Two services are being deployed on release branches to ECS:
1. reaction-core representing the Reaction app using the 'dev' MongoDB dataset
1. sk-core representing the Reaction app using the 'catalog-sample-data' MongoDB dataset; this will be used as the backend for the Storefront starterkit staging app

## Breaking changes
The URL for the Reaction staging app will change. It will be communicated internally. There will also be a new URL for the Reaction app to be used as the backend for the Storefront starterkit staging app.

## Testing
1. Verify functionality for the Reaction staging app (reaction-core against the 'dev' dataset)
2. Verify functionality for he Reaction app to be used as the backend for the Storefront starterkit staging app (sk-core against the 'catalog-sample-data' dataset)
